packages/phpactor.rb

```diff
@@ -13,8 +13,9 @@ class Phpactor < Package
   })
 
   depends_on 'composer'
-  depends_on 'php73'
-
+  php73 = `crew list installed | grep "php73" 2> /dev/null`.chomp
+  php74 = `crew list installed | grep "php74" 2> /dev/null`.chomp
+  depends_on 'php73' unless "#{php73}" != "" and "#{php74}" != ""
```

Comment:
Does this do anything funky with other operations such as `crew search` or `crew update`?  Also, if php72 or lower is installed, does the install abort with a message something like "Minimum version of php73 required".

Reply:
`crew search` interaction :
``` bash
$ crew search php
php: PHP is a popular general-purpose scripting language that is especially suited to web development.
php5: PHP is a popular general-purpose scripting language that is especially suited to web development.
php70: PHP is a popular general-purpose scripting language that is especially suited to web development.
php71: PHP is a popular general-purpose scripting language that is especially suited to web development.
php72: PHP is a popular general-purpose scripting language that is especially suited to web development.
(i) php73: PHP is a popular general-purpose scripting language that is especially suited to web development.
php74: PHP is a popular general-purpose scripting language that is especially suited to web development.
(i) phpactor: PHP completion, refactoring and introspection tool.
$ crew search php73
(i) php73: PHP is a popular general-purpose scripting language that is especially suited to web development.
$ crew search php74
php74: PHP is a popular general-purpose scripting language that is especially suited to web development.
$ crew search phpactor
(i) phpactor: PHP completion, refactoring and introspection tool.
```

Reply:
```
$ crew update
From https://github.com/skycocker/chromebrew
 * branch              master     -> FETCH_HEAD
HEAD is now at cd22336e Update ngrok from 2.3.15 to 2.3.35 (#4076)
Package lists, crew, and library updated.
Checking for package updates...

Your software is up to date.
```

Reply:
Displaying the error message si something I must handle. Working on it

Reply:
See f672c86 : it should solve all of this